http-proxy/squid.conf

```diff
@@ -0,0 +1,7162 @@
+#	WELCOME TO SQUID 3.3.8
```

Comment:
@thekondr Are we unable to install this config as part of some dependency?

Reply:
@Maxbey This config is different from the default installed. The changes allow connecting from the outside of the localhost.